com.microsoft.mrtk.core/Utilities/HandsUtils.cs

```diff
@@ -177,22 +175,19 @@ public TrackedHandJoint JointIndex
             {
                 get
                 {
-                    int nameIndex = Array.FindIndex(JointNames, IsJointName);
-                    if (nameIndex < 0)
+                    if (!Enum.TryParse<TrackedHandJoint>(Joint, out var handJointEnum))
                     {
                         Debug.LogError($"Joint name {Joint} not in TrackedHandJoint enum");
-                        return TrackedHandJoint.None;
+                        return TrackedHandJoint.Palm;
```

Comment:
Do we want to return `-1` or something otherwise invalid here?

Reply:
I have some concerns about it returning -1, but I think that can be alleviated by making these internal structs private so other classes can't access them an accidentally cause bad use cases